src/components/pages/CurriculumInfo/tabs/UnitsTab/UnitsTab.tsx

```diff
@@ -449,36 +468,36 @@ const UnitsTab: FC<UnitsTabProps> = ({ data }) => {
                     {units
                       .filter((unit) => isVisibleUnit(year, unit))
                       .map((unit, index) => {
+                        const isHighlighted = isHighlightedUnit(unit);
                         return (
                           <Card
                             key={unit.slug}
-                            $background={
-                              isHighlightedUnit(unit) ? "black" : "white"
-                            }
-                            $color={isHighlightedUnit(unit) ? "white" : "black"}
+                            $background={isHighlighted ? "black" : "white"}
+                            $color={isHighlighted ? "white" : "black"}
                             $flexGrow={"unset"}
                             $mb={32}
                             $mr={28}
                             $pb={64}
                             $position={"relative"}
                             $width={["100%", "calc(33% - 26px)"]}
                             data-testid={
-                              isHighlightedUnit(unit)
+                              isHighlighted
                                 ? "highlighted-unit-card"
                                 : "unit-card"
                             }
                           >
                             <BrushBorders
-                              color={
-                                isHighlightedUnit(unit) ? "black" : "white"
-                              }
+                              color={isHighlighted ? "black" : "white"}
                             />
-                            <OutlineHeading tag={"h3"} $fontSize={24} $mb={12}>
+                            <OutlineHeading tag={"div"} $fontSize={24} $mb={12}>
                               {index + 1}
                             </OutlineHeading>
-                            <Heading tag={"h3"} $font={"heading-7"}>
+                            <Box $font={"heading-7"}>
+                              {isHighlighted && (
+                                <VisuallyHidden>Highlighted:</VisuallyHidden>
```

Comment:
It's hard to tell here, but I think we're missing a space between `Highlighted:` and the unit title.